spec/features/reviews/create_new_review_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature 'Create a new review' do
+  let(:venue) { FactoryGirl.create(:venue) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  before do
+    visit new_venue_review_path(venue.id)
+  end
+
+  scenario 'User fills out review form correctly' do
```

Comment:
build failed with this sidenote: 

 `it` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).

tried to make scenario more descriptive, is that okay?